tests/test_WavelengthFitData.py

```diff
@@ -18,37 +18,41 @@ class WavelengthFitDataTestCase(lsst.utils.tests.TestCase):
     def setUp(self):
         self.length = 128
         self.fiberId = np.arange(123, 456, 78, dtype=np.int32)
-        self.pixels = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.nominalXPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.fitXPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.fitPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.nominalPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.fitPixelPosErr = np.random.uniform(size=self.fiberId.shape)*self.length*0.01
         self.wlMin = 600.0  # nm
         self.wlMax = 900.0  # nm
         scale = self.wlMax - self.wlMin
-        self.measuredWavelength = np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin
+        self.fitWavelength = np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin
         self.actualWavelength = np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin
-        self.lines = [LineData(*args) for args in zip(self.fiberId, self.pixels, self.measuredWavelength,
-                                                      self.actualWavelength)]
+        self.fitWavelengthErr = (np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin)*0.0
```

Comment:
Multiply by zero? Why not use `np.zeros`?

Reply:
It seems 1 (0.01) is mistakenly removed (1% Error is assigned) .